cartographer_grpc/map_builder_context.cc

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright 2018 The Cartographer Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "cartographer_grpc/map_builder_context.h"
+#include "cartographer_grpc/map_builder_server.h"
+
+namespace cartographer_grpc {
+
+MapBuilderContext::MapBuilderContext(MapBuilderServer* map_builder_server)
+    : map_builder_server_(map_builder_server) {}
+
```

Comment:
nit: Usually, getters, setters and constructors are implemented right in the header, if they are trivial. I would move them there. Up to you, however.